### ISSUE TITLE
Add proper fonts

### ISSUE THREAD

**Issue by felixjung** [sumup/circuit-ui#25]
We need to load our new font. For this we should either define `font-faces` or find some other way to declare and load fonts efficiently. Most importantly, we need to have the fonts. The URL from typekit does not seem to work.

It's possible the actual font is only relevant in the projects using this library. If those declare the font families with names this library uses, we don't need to care about this here? Do we want to buy a license for everyone using this library? I bet that does not work 😆 So probably we should keep fonts separate from circuit after all?

Potentially useful resources:
- https://medium.com/@francis.john/web-font-optimization-at-nerdwallet-6a447be9b570
- https://www.zachleat.com/web/comprehensive-webfonts/

**Comment by artsheeps** [comment]
I tried to republish the font. Can you try again?

Here the settings used:
`*.sumup.com,localhost,sumup.github.io,dashboard-beta.sam-app.ro,dashboard.sam-app.ro,me.sumup.com`

Let me know if you want me to add other domains.

**Comment by connor-baer** [comment]
In my opinion, [FOFT, or FOUT with Two Stage Render](https://www.zachleat.com/web/comprehensive-webfonts/#foft) is what we want. 

TL;DR: 
> This approach builds on the FOUT with a Class method and is useful when you’re loading multiple weights or styles of the same typeface, e.g. roman, bold, italic, bold italic, book, heavy, and others. 
> We split those web fonts into two stages: the roman first, which will then also immediately render faux-bold and faux-italic content (using font synthesis) while the real web fonts for heavier weights and alternative styles are loading.
